docs/hosting/architecture/database-structure.md

```diff
@@ -19,20 +19,36 @@ To inspect the n8n database, you can use [DBeaver](https://dbeaver.io){:target=_
 
 These are the tables n8n creates during setup.
 
+### auth_identity
+
+Stores details of external authentication providers when using [SAML](/user-management/saml/). [TODO - confirm]
+
+### auth_provider_sync_history
+
+Stores the history of a SAML connection. [TODO - confirm]
+
 ### credentials_entity
 
 Stores the credentials used to authenticate with integrations.
 
-### eventdestinations_entity
+### event_destinations
 
 Enterprise only.
```

Comment:
This table does still exist, This part may not be needed.

Reply:
I can't see it with DBeaver? It's not in the diagram it generated either?

Reply:
@StarfallProjects that is interesting, I can see it in my Postgres database. I would be surprised if we made a table when the license is populated 🤔

Reply:
Let us never speak of this again